src/tokens/formats/json-formats.js

```diff
@@ -0,0 +1,70 @@
+const jsonVariables = {
+  name: 'rvt/json/variables',
+  formatter: function(dictionary) {
+    const allProperties = dictionary.allProperties;
+    const properties = dictionary.properties;
+    // eslint-disable-next-line prefer-const
+    let categories = Object.keys(properties);
+
+    // Wrapper is the string that will be built into the final output
+    let wrapper = ``;
+    wrapper += `{\n`;
```

Comment:
I believe this could all be done in one go:

```suggestion
    let wrapper = `{\n`;
```